tests/integration/steps.go

```diff
@@ -1,512 +1,28 @@
-				},
-			},
-			chainID("consu"): ChainState{
-				ValPowers: &map[validatorID]uint{
-					validatorID("alice"): 509,
-					validatorID("bob"):   0,
-					validatorID("carol"): 501,
-				},
-			},
-		},
-	},
-	{
-		action: relayPacketsAction{
-			chain:   chainID("provi"),
-			port:    "provider",
-			channel: 0,
-		},
-		state: State{
-			chainID("consu"): ChainState{
-				ValPowers: &map[validatorID]uint{
-					validatorID("alice"): 509,
-					validatorID("bob"):   495,
-					validatorID("carol"): 501,
-				},
-			},
-		},
-	},
-	// Now we test provider initiated downtime/slashing
-	{
-		action: downtimeSlashAction{
-			chain:     chainID("provi"),
-			validator: validatorID("carol"),
-		},
-		state: State{
-			chainID("provi"): ChainState{
-				ValPowers: &map[validatorID]uint{
-					// Non faulty validators still maintain just above 2/3 power here
-					validatorID("alice"): 509,
-					validatorID("bob"):   495,
-					validatorID("carol"): 0,
-				},
-			},
-			chainID("consu"): ChainState{
-				ValPowers: &map[validatorID]uint{
-					validatorID("alice"): 509,
-					validatorID("bob"):   495,
-					validatorID("carol"): 501,
-				},
-			},
-		},
-	},
-	{
-		action: relayPacketsAction{
-			chain:   chainID("provi"),
-			port:    "provider",
-			channel: 0,
-		},
-		state: State{
-			chainID("consu"): ChainState{
-				ValPowers: &map[validatorID]uint{
-					validatorID("alice"): 509,
-					validatorID("bob"):   495,
-					validatorID("carol"): 0,
-				},
-			},
-		},
-	},
-	{
-		action: unjailValidatorAction{
-			provider:  chainID("provi"),
-			validator: validatorID("carol"),
-		},
-		state: State{
-			chainID("provi"): ChainState{
-				ValPowers: &map[validatorID]uint{
-					validatorID("alice"): 509,
-					validatorID("bob"):   495,
-					validatorID("carol"): 495,
-				},
-			},
-			chainID("consu"): ChainState{
-				ValPowers: &map[validatorID]uint{
-					validatorID("alice"): 509,
-					validatorID("bob"):   495,
-					validatorID("carol"): 0,
-				},
-			},
-		},
-	},
-	{
-		action: relayPacketsAction{
-			chain:   chainID("provi"),
-			port:    "provider",
-			channel: 0,
-		},
-		state: State{
-			chainID("consu"): ChainState{
-				ValPowers: &map[validatorID]uint{
-					validatorID("alice"): 509,
-					validatorID("bob"):   495,
-					validatorID("carol"): 495,
-				},
-			},
-		},
-	},
+func concatSteps(steps ...[]Step) []Step {
+	var concat []Step
+	for _, s := range steps {
+		concat = append(concat, s...)
+	}
+	return concat
+}
 
-	// TODO: Test full unbonding functionality, tracked as: https://github.com/cosmos/interchain-security/issues/311
+var happyPathSteps = concatSteps(
+	stepsStartChains("consu", "", false),
+	stepsDelegate("consu"),
+	stepsUnbondRedelegate("consu"),
+	stepsDowntime("consu"),
+)
 
-}
+var democracySteps = concatSteps(
+	// democracySteps requires a transfer channel and overrides genesis
+	stepsStartChains("democ", ".app_state.ccvconsumer.params.blocks_per_distribution_transmission = \"10\"", true),
```

Comment:
Imo there should be a single place that the genesis file is mutated, lets consolidate this with the way of mutating genesis in `config.go`

Reply:
Agree, moved the string to config.go for democracy chain.